src/kv/RocksDBStore.cc

```diff
@@ -1448,7 +1448,7 @@ void RocksDBStore::get_statistics(Formatter *f)
       f->close_section();
     }
     f->open_object_section("rocksdbstore_perf_counters");
-    logger->dump_formatted(f,0);
+    logger->dump_formatted(f,0, false);
```

Comment:
Nice to fix bad style when touching the line anyway:
```suggestion
    logger->dump_formatted(f, false, false);
```